### DISCUSSION TITLE
Incremented indents instead of consistent indents.

### DISCUSSION THREAD

**Discussion by kostia-Etaplus** [community/community#113424] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I've noticed a bug that whenever github copilot generates code with concatenation of strings via `+` separated by a new line or multiple method arguments also with new lines in between, it increments indent size for every new line. Quite annoying thing. So for instance instead of this code sample:

```
    "I'm a first line.\n" +
    "I'm a second line.\n" +
    "I'm a third line."

```
github copilot would generate this code:

```
    "I'm a first line.\n" +
        "I'm a second line.\n" +
            "I'm a third line."
```
If the first indent is bigger, then following also are proportially bigger (like x2, x3 and so on, often it's somewhere far outside a screen).

I'm using Windows 11, Visual Studio 2022 community (64-bit) version 17.8.2.
Mostly coding in C#.

**Comment by Akash1134** [comment]
Closing this as a duplicate of [this discussion](https://github.com/orgs/community/discussions/113425). Thanks!
